config/zones/CH.yaml

```diff
@@ -73,11 +73,30 @@ emissionFactors:
         source: Electricity Maps, 2021 average
         value: 113.07156762843732
     unknown:
-      _comment:
-        'Source ENTSOE, Swiss government and Electricity Maps: https://github.com/electricitymaps/electricitymaps-contrib/pull/2898
-        and https://docs.google.com/spreadsheets/d/1FLHQ6e9Es08BIqX654BM3SEb_fuAk4k4O-6cmRXyw_E/'
-      source: 52% hydro storage, 48% unknown thermal
-      value: 288
+      - datetime: '2017-01-01'
+        _comment: 'Assumed make up of the unknown production is detailed here: https://docs.google.com/spreadsheets/d/1FLHQ6e9Es08BIqX654BM3SEb_fuAk4k4O-6cmRXyw_E'
```

Comment:
Are we sure the unknown renewables are hydro? They currently have a 24 gCO2eq/kWh.

Reply:
Well no .. but that's my assumption. It's unclear what it is, we just know it's "unknown renewable"

Reply:
I think it's fine for now as we haven't determined the exact composition. But it's likely it's not 100% hydro.